### DISCUSSION TITLE
브로커의 토픽을 구성할 때 파티션은 정족수를 만족해야 하는 지 궁금합니다

### DISCUSSION THREAD

**Discussion by seongtaekkim** [kyungjunleeme/kafka_study#30] [Q&A]
브로커의 토픽을 구성할 때 파티션은 정족수를 만족해야 하는 지 궁금합니다

**Comment by kyungjunleeme** [comment]
질문 내용의 주체가 파티션 자체가 정족수를 만족해야하는 것에 대한 답은 아니지만, 관련 있는 부분이라서 남깁니다. 


![image](https://github.com/user-attachments/assets/e070d303-0e32-45ea-a83d-7828e3fc0464)
![image](https://github.com/user-attachments/assets/a0b03431-83a9-49d6-a345-ae4f92480d7a)
![image](https://github.com/user-attachments/assets/0a389edd-a595-4d41-969a-446c644af06e)
